dali/operators/reader/numpy_reader_op.cc

```diff
@@ -45,7 +47,7 @@ static void CopyHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &inp
   }
 }
 
-static void TransposeHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &input) {
+static void TransposeHelper(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input) {
```

Comment:
Helperpocalypse continued....
```suggestion
static void ReverseStorageOrder(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input) {
```

Reply:
Do we really need to inflate the number of comments for this reason :upside_down_face: